BrowserKit/Package.swift

```diff
@@ -33,6 +33,9 @@ let package = Package(
         .library(
             name: "MenuKit",
             targets: ["MenuKit"]),
+        .library(
```

Comment:
FYI @adudenamedruby — UnifiedSearchKit is just MenuKit code with renamed types and stripped functionality (hopefully that makes the review easier!). The only change is that the Element contains a UIImage instead of an asset name.